test/helper/test_timeline.js

```diff
@@ -23,16 +23,18 @@ const setup = (timelineFiles) => {
   timelineFiles.forEach((timelineFile) => {
     // For each timeline file, ensure that its corresponding trigger file is pointing to it.
     const filename = timelineFile.split('/').slice(-1)[0];
-    const triggerFilename = timelineFile.replace('.txt', '.js');
-    if (!fs.statSync(triggerFilename))
+    const triggerFilenameJS = timelineFile.replace('.txt', '.js');
+    const triggerFilenameTS = timelineFile.replace('.txt', '.ts');
+    if (!fs.existsSync(triggerFilenameJS) && !fs.existsSync(triggerFilenameTS))
```

Comment:
Can you store the results of existsSync in a variable? I know it's premature optimization, but checking if a file exists on disk is probably the most expensive operation here.